src/psyclone/tests/gocean1p0_test.py

```diff
@@ -288,6 +288,60 @@ def test_scalar_float_arg(tmpdir):
     assert GOcean1p0Build(tmpdir).code_compiles(psy)
 
 
+def test_scalar_float_arg_from_module(tmpdir):
```

Comment:
Could use `pytest.mark.usefixtures` to avoid pylint warning about unused tmpdir, if in fact, tmpdir is needed at all.